data/virt_autotest/sriov_network_guest_logging.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+#xen irqbalance guest debugging
+date
+set -x
```

Comment:
quick question, is there still need some debug info from call` 'set -x'` ?

Reply:
'set -x' will print the command itself rather than the output only, then we can know what the output is from.